events_types.go

```diff
@@ -214,11 +214,19 @@ type EventCrashType struct {
 }
 
 func (ev *EventCrashType) String() string {
-	return fmt.Sprintf("container '%v' has crashed", ev.TypeData.ContainerType)
+	msg := fmt.Sprintf("container '%v' has crashed", ev.TypeData.ContainerType)
+
+	if ev.TypeData.CrashLogs != "" {
+		dashboard_url := "https://my.scalingo.com/apps/" + ev.AppName + "/events/" + ev.ID
```

Comment:
Yes I totally agree with @johnsudaar, only the API should have the domain name configuration for the dashboard etc.

Moreover `dashboard_url` -> `dashboardURL`, come on you're coding with Go!